RecoTracker/TrackProducer/test/testProbQXY.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+function die { echo $1: status $2 ; exit $2; }
+
+echo "TESTING Getting probQ and probXY from Analysis Data-Tier codes ..."
+echo "For now not doing anything, when there will be AOD and MiniAOD with this variable, this can be uncommented"
+# test AOD
+# Will come soon
+#echo "TESTING refitting from AOD ...\n\n"
+#cmsRun ${LOCAL_TEST_DIR}/ProbQXYFromMiniAOD.py maxEvents=100 inputFiles=/store/mc/RunIISummer20UL16RECO/DYJetsToMuMu_M-50_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/AODSIM/106X_mcRun2_asymptotic_v13-v2/00000/1D4FBF4B-7B8D-A04F-A108-B1BEB60558FA.root || die "Failure refitting from AOD" $?
+
+# test MINIAOD
+#echo "TESTING  Getting probQ and probXY from MINIAOD ...\n\n"
+#cmsRun ${LOCAL_TEST_DIR}/ProbQXYFromMiniAOD.py maxEvents=100 inputFiles=/store/mc/RunIISummer20UL16MiniAOD/DYJetsToMuMu_M-50_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/MINIAODSIM/106X_mcRun2_asymptotic_v13-v2/00000/01916D91-A314-D947-A420-388891D62FEA.root || die "Failure getting probQ and probXY from MINIAOD" $?
```

Comment:
this is already implemented, right?

Reply:
Yes, but the input data doesnt exist yet

Reply:
IIUC if a unit test is added consuming a given RAW data file, it will remain staged, and if it ever gets removed it can be copied to the cmsuser core eos area.
Can you comment @smuzaffar ?

Reply:
correct @mmusich , this file has already been cached in ibeos area `/eos/cms/store/user/cmsbuild/store/mc/RunIISummer20UL16MiniAOD/DYJetsToMuMu_M-50_TuneCP5_13TeV-powhegMiNNLO-pythia8-photos/MINIAODSIM/106X_mcRun2_asymptotic_v13-v2/00000/01916D91-A314-D947-A420-388891D62FEA.root`  and will remain on disk as long as any unit test or workflow access it.

Reply:
@smuzaffar I'd need root://cms-xrd-global.cern.ch//store/data/Run2018C/SingleMuon/RAW/v1/000/319/337/00000/004ED286-5582-E811-8895-FA163E126126.root to be cached too, thanks!

Reply:
@tvami done https://cmssdt.cern.ch/jenkins/job/lfn-to-ibeos/1086/console